### ISSUE TITLE
Problem: role not compatible with python3

### ISSUE THREAD

**Issue by mamedin** [artefactual-labs/ansible-nginx#11]
Role fails when using ansible and python3:
``` 
TASK [external-roles/artefactual-nginx : Ensure auth_basic files created] *****************************************************************************************************************************************
failed: [XXX.accesstomemory.org] (item=dict_keys([])) => {"ansible_loop_var": "item", "changed": false, "item": "dict_keys([])", "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'dict_keys([])'"}
```

**Comment by mamedin** [comment]
See: https://docs.ansible.com/ansible/2.4/playbooks_python_version.html#dictionary-views

"In Python2, the dict.keys(), dict.values(), and dict.items() methods returns a list. Jinja2 returns that to Ansible via a string representation that Ansible can turn back into a list. In Python3, those methods return a dictionary view object. The string representation that Jinja2 returns for dictionary views cannot be parsed back into a list by Ansible. It is, however, easy to make this portable by using the list filter whenever using dict.keys(), dict.values(), or dict.items()."

**Comment by hakamine** [comment]
Fixed by #10 (same as #12)
